tests/Rules/Doctrine/ORM/EntityColumnRuleTest.php

```diff
@@ -38,6 +40,12 @@ protected function getRule(): Rule
 		if (!Type::hasType(UuidType::NAME)) {
 			Type::addType(UuidType::NAME, UuidType::class);
 		}
+		if (!Type::hasType(CarbonType::NAME)) {
+			Type::addType(CarbonType::NAME, \Carbon\Doctrine\CarbonType::class);
```

Comment:
the `Carbon\Doctrine\CarbonType` doesn't have a NAME constant, so I added it [here](https://github.com/phpstan/phpstan-doctrine/pull/192/files#diff-8c4cceaaf5869a43f944547773744cf916f09124dfb9eaf6fe6290bb83f2e38aR13) instead